### DISCUSSION TITLE
Unable to import component: cannot import name 'make_byte_string'

### DISCUSSION THREAD

**Discussion by nuki47** [binsentsu/home-assistant-solaredge-modbus#175] [Q&A]
Sometimes I get this error:

Setup failed for custom integration solaredge_modbus: Unable to import component: cannot import name 'make_byte_string' from 'pymodbus.utilities' (/usr/local/lib/python3.11/site-packages/pymodbus/utilities.py)

After a reboot it works for a while...

**Comment by Breina** [comment]
Pretty sure this one can get closed. Had the same issue on my repo and it was fixed by upgrading the `pymodbus` requirement in the manifest, which you guys already did.
